plexus.config.example

```diff
@@ -21,3 +21,7 @@ PLEXUS_LOG_LEVEL=info
 
 # Log everything to a file.
 PLEXUS_LOG_FILE=
+
+# Plexus auth for creating sessions
+PLEXUS_AUTH_USER="admin"
+PLEXUS_AUTH_PASSWORD="foobaz"
```

Comment:
This example file should contain the default values, could you change the user & password to an empty string? Also it would be great if we can rename password to pass, then it will be consistent with MESH_CENTRAL_PASS.
```suggestion
PLEXUS_AUTH_USER=
PLEXUS_AUTH_PASS=
```